lib/mongo/collection/view/readable.rb

```diff
@@ -150,6 +152,67 @@ def count(opts = {})
           end
         end
 
+        # Get a count of matching documents in the collection.
```

Comment:
"Get the number of matching documents" or "count the matching documents" I think would be more grammatically correct.